dev/tools/gen_date_localizations.dart

```diff
@@ -77,7 +77,7 @@ Future<Null> main(List<String> rawArgs) async {
 
 // This file has been automatically generated.  Please do not edit it manually.
 // To regenerate run (omit -w to print to console instead of the file):
-// dart dev/tools/gen_date_localizations.dart -w
+// dart -c dev/tools/gen_date_localizations.dart -w
```

Comment:
Isn't `-c` going away in Dart 2?

Reply:
Yeah, it does. Replaced it with `--enable-asserts`